GitHub.Collectors.Functions/GitHubFunctions.cs

```diff
@@ -652,6 +652,97 @@ public async Task Traffic([QueueTrigger("traffic")] string queueItem, ExecutionC
             }
         }
 
+        [FunctionName("PointCollector")]
+        public Task PointCollector([QueueTrigger("point")] string queueItem, ExecutionContext executionContext, ILogger logger, int dequeueCount)
+        {
+            return this.ExecutePointCollectorAsync(queueItem, executionContext, logger, dequeueCount);
+        }
+
+        public async Task ExecutePointCollectorAsync(string queueItem, ExecutionContext executionContext, ILogger logger, int dequeueCount)
+        {
+            DateTime functionStartDate = DateTime.UtcNow;
+            string sessionId = Guid.NewGuid().ToString();
+
+            JsonSerializerSettings serializerSettings = new JsonSerializerSettings()
+            {
+                TypeNameHandling = TypeNameHandling.None
+            };
+            PointCollectorInput pointCollectorInput = JsonConvert.DeserializeObject<PointCollectorInput>(queueItem, serializerSettings);
+
+            FunctionContext context = new FunctionContext()
+            {
+                CollectorType = CollectorType.Point.ToString(),
+                CollectorIdentity = this.configManager.GetCollectorIdentity(),
+                FunctionStartDate = functionStartDate,
+                SessionId = sessionId,
+                InvocationId = executionContext.InvocationId.ToString(),
+                DequeueCount = dequeueCount,
+            };
+
+            ITelemetryClient telemetryClient = new GitHubApplicationInsightsTelemetryClient(this.telemetryClient, context, logger);
+            bool success = false;
+            string outputPaths = String.Empty;
+            Dictionary<string, string> sessionEndProperties = new Dictionary<string, string>();
+            StatsTracker statsTracker = null;
+            string identifier = "Point";
+            try
+            {
+                ICache<PointCollectorTableEntity> pointCache = new AzureTableCache<PointCollectorTableEntity>(telemetryClient, "point");
+                await pointCache.InitializeAsync().ConfigureAwait(false);
+                Repository repository = pointCollectorInput.getRepository();
+                ICache<RateLimitTableEntity> rateLimiterCache = new AzureTableCache<RateLimitTableEntity>(telemetryClient, "ratelimiter");
+                await rateLimiterCache.InitializeAsync().ConfigureAwait(false);
+                IRateLimiter rateLimiter = new GitHubRateLimiter(this.configManager.UsesGitHubAuth(context.CollectorType) ? repository.OrganizationLogin : "*", rateLimiterCache, this.httpClient, telemetryClient, maxUsageBeforeDelayStarts: 95.0, this.apiDomain, throwOnRateLimit: true);
```

Comment:
Let's not give 95% of the usage to point collector, at least until all web hook processing is off webhook processor. Maybe start with 75-85%?